src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  margin: 0 auto;
```

Comment:
BEM-block shouldn't have any margins, it you want to specify them use BEM mix instead.